server-setup/termux-installation

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+nav_order: [[no idea what to set this to]]
```

Comment:
nav_order is the order in which sections appear on the sidebar. If you don't want to move the other sections, just set it to 4.